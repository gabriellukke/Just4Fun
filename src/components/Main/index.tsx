const Main = () => (
  <main>
    <h1>React Next Boilerplate</h1>
  </main>
);

export default Main;
