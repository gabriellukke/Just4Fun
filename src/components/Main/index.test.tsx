import { render, screen } from '@testing-library/react';

import Main from '.';

describe('<Main/>', () => {
  it('should render the header', () => {
    const { container } = render(<Main />);

    const heading = screen.getByRole('heading', {
      name: /react next boilerplate/i,
    });
    expect(heading).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
