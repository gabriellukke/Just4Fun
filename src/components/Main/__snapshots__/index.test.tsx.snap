// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main/> should render the header 1`] = `
<main>
  <h1>
    React Next Boilerplate
  </h1>
</main>
`;
